paramspider.py

```diff
@@ -114,9 +114,6 @@ def main():
         else:
             black_list.append("." + args.exclude)
 
-    else:
-        black_list = []  # for blacklists
```

Comment:
We don't need this equation because we already set black_list to empty list in line 108.